<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AKPOL synchronizacja</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/app/icons/app/Treetog-Junior-Sync.ico</normaloff>:/app/icons/app/Treetog-Junior-Sync.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>Wykryte napędy typu pendrive</string>
        </property>
        <property name="text">
         <string>Pendrive'y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="removableMedias">
        <property name="toolTip">
         <string>Wykryte napędy typu pendrive</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="scanAndCopy">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Skanuj wszystkie pendrive'y i wyślij archiwa WF-MAG</string>
      </property>
      <property name="text">
       <string>SKANUJ i WYŚLIJ WF-MAG</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/app/icons/app/Treetog-Junior-Sync.ico</normaloff>:/app/icons/app/Treetog-Junior-Sync.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionSelectRemovables"/>
    <addaction name="actionSelectDest"/>
    <addaction name="actionSetTimeout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionSelectDest">
   <property name="text">
    <string>Wybierz katalog docelowy</string>
   </property>
  </action>
  <action name="actionSelectRemovables">
   <property name="text">
    <string>Wybierz pendrive'y</string>
   </property>
   <property name="toolTip">
    <string>Wybierz pendrive'y do skanowania</string>
   </property>
  </action>
  <action name="actionSetTimeout">
   <property name="text">
    <string>Ustaw okres skanowania</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
